restcomm.android.client.sdk/src/main/java/org/mobicents/restcomm/android/client/sdk/RCDevice.java

```diff
@@ -370,6 +370,10 @@ public RCConnection connect(Map<String, Object> parameters, RCConnectionListener
             connection.state = RCConnection.ConnectionState.PENDING;
             DeviceImpl.GetInstance().sipuaConnectionListener = connection;
 
+            if (enableVideo == null) {
```

Comment:
@atsakiridis The issue is when "video-enabled" is not present, the enableVideo is null and enableVideo.booleanValue() results in NPE.